release/README.md

```diff
@@ -2,253 +2,31 @@
-
-```
-$ git clone https://github.com/istio/test-infra.git
-```
-
-and build it using
-
-```
-$ bazel build //toolbox/githubctl
-```
-
-The binary output is located in bazel-bin/toolbox/githubctl/githubctl.
-
-```
-$ alias githubctl="${PWD}/bazel-bin/toolbox/githubctl/githubctl"
-```
-
-The release process goes like the following:
-
-### Step 1: Tag the release.
-```
-$ githubctl --token_file=<github token file> \
-    --op=tagIstioDepsForRelease \
-    --base_branch=<release branch or master>
-```
-
-### Step 2: The previous command triggers rebuild and retagging on pilot, proxy, mixer and auth(security).
- Wait for them to finish. Check build job status [here](https://console.cloud.google.com/gcr/builds?project=istio-io&organizationId=433637338589).
-
-### Step 3: Create an update PR in istio/istio.
-```
-$ githubctl --token_file=<github token file> \
-    --op=updateIstioVersion --base_branch=<release branch or master>
-```
-This will run all the presubmits on the istio repo, smoke testing the created artifacts.
-
-### Step 4: Request PR approval and wait for the PR to be merged. Note down the SHA
-of the merged PR in `RELEASE_SHA`. We will create the release tag from it.
-
-### Step 5: Finalize the release. 
-This creates the release draft in GitHub, uploads the artifacts,
-and creates a PR to advances next release tag, and updates [download script](https://github.com/istio/istio/blob/master/release/downloadIstioCandidate.sh) with latest release:
- 
-```
-$ githubctl --token_file=<github token file> \
-    --op=uploadArtifacts --base_branch=<release branch or master> \
-    --next_release=<next release> --ref_sha=${RELEASE_SHA}
-```
-
-Note: 
-
-0. If you are cutting a release off of **a release branch other than master**, you could have the `downloadIstioCandidate.sh` script in master branch to take on the release you just made by also specifying the flag `--update_rel_branches=master`.
-1. `<next release>` is where the next release after the release draft that is being created.  For example, if you are creating 0.2.7 release, the next release could be 0.2.8.  
-2. For Mac, install `gcp` via ```brew install coreutils``` and install `gtar` via ```brew install gnu-tar```.  Execute the command below instead:
-
-```
-$ TAR=gtar CP=gcp githubctl --token_file=<github token file> \
-    --op=uploadArtifacts --base_branch=<release branch or master> \
-    --next_release=<next release> --ref_sha=${RELEASE_SHA}
-```
-
-You can finalize release in github release [page](https://github.com/istio/istio/releases) before this PR got merged, but it's your responsibility to make sure it got merged for next release. **You should finalize it as an actual release (not shown as "draft") before Step 6**
- 
-### Step 6: Generating release note.
```

Comment:
The release note step hasn't been automated yet so it's still a manual step.  As of yesterday I did complete an initial port of the functionality to bash though I'm not sure if we want to start using it at this phase yet.  The tool might just be "best effort" and if there's any issues or problems (or someone wants to customize the range of time to query for the notes) then someone might want to run the tool instead.

Reply:
The goal is to reduce the chance someone outside of the test/release group "might want to run the tool" for now, until the process (and probably the UI) is ready for external use.